src/format_text.rs

```diff
@@ -137,7 +138,7 @@ fn build_json_text(formatted_text: &str, indent_text: &str) -> String {
         if is_last_line {
           Cow::Borrowed(line)
         } else {
-          Cow::Owned(format!("{}\n", line))
+          Cow::Owned(format!("{line}\n"))
```

Comment:
I've found that string interpolation in Rust to not be good. Here's my copy and pasted reasons from a chat I was having:

> 1. The inline version is limited. Ex. `something.display()` doesn't work. So when it doesn't work you have to revert back to the non-inline version. Might as well just use non-inline all the time then
> 2. The tooling with rust analyzer is not good atm. Renames don't happen and there's no syntax highlighting
> 3. It's easy to accidentally take a string and make it a non-format string and not realize (especially since there's no syntax highlighting). Then you have {something} in your output instead of what's in the something variable. Impossible to do this with the non-inline version.

Reply:
since rust-analyzer now [supports implicit format args](https://github.com/rust-lang/rust-analyzer/pull/16027), could this be reconsidered?